lib/traject/pdc_describe_indexing_config.rb

```diff
@@ -4,7 +4,6 @@
 require 'traject'
 require 'traject/nokogiri_reader'
 require 'blacklight'
-require_relative './domain'
```

Comment:
Might this file be used elsewhere? If not, I just wished to please confirm that removing this file from the repository entirely was not preferred before I merge this.